README.md

```diff
@@ -99,23 +99,28 @@ Here is a description of the events:
 ```js
 import CallingExtensions from "@hubspot/calling-extensions-sdk";
 
+/**
+ * Whether to log various inbound/outbound debug messages to the console.
+ * @typedef {Object} Options
+ * @property {boolean} debugMode - Whether to log various inbound/outbound debug messages to the console. If false, console.debug will be used instead of console.log.
+ * @property {Object} eventHandlers - Handle inbound messages
+ */
+
+/**
+ * @typedef {Object} EventHandlers
+ * @property {function} onReady - Called when HubSpot is ready to receive messages.
+ * @property {function} onDialNumber - Called when HubSpot sends a dial number from the contact.
+ * @property {function} onEngagementCreated - Called when HubSpot creates an engagement for the call.
+ * @property {function} onVisibilityChanged - Called when the call widget's visibility changes.
+ */
+
```

Comment:
@alonso-cadenas What do you think of using jsdocs to document the options object here?